bin/security/setupSecurityAccount.sh

```diff
@@ -2,20 +2,21 @@
 
 option="${1}"
 
+# x86_64 or arm64 for make run option
+[ "$(uname -m)" != "x86_64" ] && USE_ARM64="arm64"
+
 BB_ROOT_DIR=$(cd $(dirname "$0"); cd ..; pwd)
-COMPOSE=$BB_ROOT_DIR/$(ls $BB_ROOT_DIR | awk '/docker-compose/ && !/test-tools/')
+MAKEFILE=$BB_ROOT_DIR/Makefile
 
 case ${option} in
   -useradd)
   echo "Info: Add security account."
-  OT=$(docker-compose -f ${COMPOSE} run --rm \
-    --entrypoint /edgex/security-proxy-setup edgex-proxy --init=false --useradd=geneva --group=admin | grep '^the access token for')
-  export TOKEN=$(echo ${OT} | sed 's/.*: \([^.]*\.[^.]*\.[^.]*\).*/\1/')
+  export TOKEN=`make -f ${MAKEFILE} get-token ${USE_ARM64} geneva`
```

Comment:
is the version word like `geneva` supposed to be set in env. var? or we have to change this from version to version. or this is just an example to be more generalized?

Reply:
The user name is opaque to the client.  But maybe it should say 'testinguser' or something.

Reply:
I agree that `geneva` isn't appropriate, a bit beyond scope of this PR, but went ahead and changed to `testinguser`.